examples/platform_channel_swift/ios/Runner/AppDelegate.swift

```diff
@@ -19,7 +19,7 @@ enum MyFlutterErrorCode {
   static let unavailable = "UNAVAILABLE"
 }
 
-@UIApplicationMain
+@main
```

Comment:
This file contains significant edits from the starter app template. The migration is flexible enough to be able to update this correctly.